webapp/lib/logger.dart

```diff
@@ -31,6 +31,10 @@ trace(String f, String s) {
   }
 }
 
+severe(String s) {
```

Comment:
We might also consider extending the API of this with e.g. LogException

Reply:
I think I would like to add some more structure to our logging to make parsing the logs afterwards easier

Reply:
That sounds good. I haven't done anything about this yet, but I'll open an issue about it and we'll do it in a different PR